app/javascript/dashboard/modules/search/components/SearchView.vue

```diff
@@ -1,46 +1,62 @@
 <template>
-  <section class="search-root">
-    <woot-button
-      color-scheme="secondary"
-      size="large"
-      icon="dismiss"
-      variant="smooth"
-      class="modal--close"
-      @click="onBack"
-    />
-    <header>
-      <search-header @search="search" />
-      <search-tabs :tabs="tabs" @tab-change="tab => (selectedTab = tab)" />
-    </header>
-    <div class="search-results">
-      <woot-loading-state v-if="uiFlags.isFetching" :message="'Searching'" />
-      <div v-else>
-        <div v-if="all.length">
-          <search-result-contacts-list
-            v-if="filterContacts"
-            :contacts="contacts"
-            :query="query"
-          />
-          <search-result-messages-list
-            v-if="filterMessages"
-            :messages="messages"
-            :query="query"
-          />
-          <search-result-conversations-list
-            v-if="filterConversations"
-            :conversations="conversations"
-            :query="query"
-          />
-        </div>
-        <div v-else-if="showEmptySearchResults" class="empty">
-          <fluent-icon icon="info" size="16px" class="icon" />
-          <p class="empty-state__text">
-            {{ $t('SEARCH.EMPTY_STATE_FULL', { query }) }}
-          </p>
+  <div class="backdrop">
+    <section v-on-clickaway="closeSearch" class="search-root">
+      <woot-button
+        color-scheme="secondary"
+        size="large"
+        icon="dismiss"
+        variant="smooth"
+        class="modal--close"
+        @click="onBack"
+      />
+      <header>
+        <search-header @search="search" />
+        <search-tabs
+          v-if="!showEmptySearchResults && all.length"
+          :tabs="tabs"
+          @tab-change="tab => (selectedTab = tab)"
+        />
+      </header>
+      <div class="search-results">
+        <woot-loading-state v-if="uiFlags.isFetching" :message="'Searching'" />
+        <div v-else>
+          <div v-if="all.length">
+            <search-result-contacts-list
+              v-if="filterContacts"
+              :contacts="contacts"
+              :query="query"
+              @close-search="closeSearch"
+            />
+            <search-result-messages-list
+              v-if="filterMessages"
+              :messages="messages"
+              :query="query"
+              @close-search="closeSearch"
+            />
+            <search-result-conversations-list
+              v-if="filterConversations"
+              :conversations="conversations"
+              :query="query"
```

Comment:
```suggestion
              :query="query"
              @close-search="closeSearch"
```

Seems like you missed the listener here